_templates/countdown.php

```diff
@@ -0,0 +1,9 @@
+<script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
```

Comment:
Yeah I noted this in the PR comments. For some reason jquery isn't being loaded until later, maybe. I dunno. It's not working though.